app/javascript/packs/stylesheets/bootstrap.scss

```diff
@@ -195,10 +118,49 @@ select.form-control {
   font-weight: 250 !important;
 }
 
+.nav-tabs .nav-link {
+  border-top-left-radius: 0 !important;
+  border-top-right-radius: 0 !important;
+}
+
+.nav-tabs .nav-link:hover {
+  border-color:#b5b5b5;
+  background-color:white;
+}
+
 .nav-link-inactive {
   color: rgba(255, 255, 255, 0.8) !important;
 }
 
+/* styling for navbar on smaller screens only */
+@media only screen and (max-width : 1199px) {
+  .nav-tabs .nav-link {
+    width: 190px;
```

Comment:
This width is not enough for the Records Requiring Review tab in isolation:

![image](https://user-images.githubusercontent.com/8235974/121824384-e761fd80-cc79-11eb-9570-27853b141660.png)

Looks like it would have to be 265px in order to fit. But this of course also makes most of the other tabs unnecessarily wide which looks kinda weird, but maybe it's fine:

![image](https://user-images.githubusercontent.com/8235974/121824439-2db75c80-cc7a-11eb-9440-80908530f7f5.png)
 
The alternative would be to `auto` width. But this makes it less "grid-like", which conflicts with the mock-ups.

![image](https://user-images.githubusercontent.com/8235974/121824498-956da780-cc7a-11eb-9b8d-aff1b330d7b0.png)

Might be worth getting Mike Nosal's opinion.

Reply:
I originally had the latter and ran this by @mrnosal.  he said they should be fixed width, but maybe I can have them be different widths in iso and exp. will look into it

Reply:
I changed this to have the width depend on exposure versus isolation.  lmk what you think

Reply:
Looks go to me